.circleci/config.yml

```
@@ -45,6 +45,10 @@ jobs:
 workflows:
   ci:
     jobs:
+      - build:
+          name: "ruby3-2_rails7-0"
+          ruby_version: 3.2.0
+          rails_version: 7.0.3
```

What about bumping the rails version to the latest?  It would at least kick off another build to see if the nokogiri issue gets resolved like in QA. :shrug: 
```suggestion
          rails_version: 7.0.4
```

---

oh nice, catch, sure, we should do that either way.

---

If that changes it... I wonder if that means it was an issue with how we do gem caching in our circleCI orb, like maybe changing rails version broke the cache... 

but if the builds never get newly released versions of gems unless rails version changes, that's actually not great for CI on several grounds (you will not be testing against actual latest dependencies).